delegation-toolkit/get-started/delegation-quickstart.md

```diff
@@ -122,7 +134,7 @@ const signedDelegation = {
 
 ### 7. Redeem the delegation
 
-The delegate account can now [redeem the delegation](../how-to/redeem-delegation.md). The redeem transaction is sent to the `DelegationManager` contract, which validates the delegation and executes actions on the delegator's behalf.
+Bob can now [redeem the delegation](../how-to/redeem-delegation.md). The redeem transaction is sent to the `DelegationManager` contract, which validates the delegation and executes actions on Alice's behalf.
```

Comment:
Since we are sending the transaction to delegateSmartAccount, we should explain how `DelegationManager` validates and execute it?